pkg/authorization/generated/internalclientset/typed/authorization/internalversion/scheme_expansion.go

```diff
@@ -0,0 +1,19 @@
+package internalversion
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api/install"
+
+	authorizationv1 "github.com/openshift/origin/pkg/authorization/apis/authorization/v1"
+	authorization "github.com/openshift/origin/pkg/authorization/generated/internalclientset/scheme"
+)
+
+func init() {
+	// This is needed in order to be able to convert the PodSpec which is part of
+	// DeploymentConfig template.
+	kapi.Install(authorization.GroupFactoryRegistry, authorization.Registry, authorization.Scheme)
```

Comment:
neither authorization nor user should require these. what tests were failing that this fixed?